routers/cart-router.js

```diff
@@ -0,0 +1,48 @@
+const express = require("express");
```

Comment:
keep db function and routing separate.

Reply:
Can you elaborate here please? I don't understand.

Reply:
i'm saying keep routing regarding functionality here, and call db function, or logical function, with contain logic, put db regarding function in model file you created.

Reply:
I'll add the db related operations in a separate folder called controllers once I refactor the code for auth. I understood what you meant. Sorry for the late reply.